packages/main/src/TableSelection.ts

```diff
@@ -72,6 +79,7 @@ class TableSelection extends UI5Element implements ITableFeature {
 	selected!: string;
 
 	_table?: Table;
+	_rangeSelection?: {selected: boolean, isUp: boolean | null, rows: TableRow[], isMouse: boolean} | null;
```

Comment:
I feel like you need a direction -1/+1 instead of isUp but need to check the code.
also why we need row instances is start/end index not enough?